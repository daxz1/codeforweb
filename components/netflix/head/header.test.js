/* global describe, it, expect */
import React from 'react';
import renderer from 'react-test-renderer';
import Header from './header';

describe('[RENDER] components/netflix/head/header.js', () => {
  it('Header Component renders', () => {
    const component = renderer.create(<Header />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
