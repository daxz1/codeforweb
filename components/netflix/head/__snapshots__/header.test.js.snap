// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[RENDER] components/netflix/head/header.js Header Component renders 1`] = `null`;
